src/content/docs/fr/reference/errors/locals-reassigned.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: locals ne doit pas être réaffecté.
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+import DontEditWarning from '~/components/DontEditWarning.astro'
```

Comment:
Maybe remove L6 to 8 because not necessary to have this component.

Reply:
Thanks! Should be right now.